File: pixo-c#/DalSql/models/Customer.cs
using System;
using System.Collections.Generic;

namespace DalSql.models;

public partial class Customer
{
    public int Id { get; set; }

    public string? FullName { get; set; }

    public string? Email { get; set; }

    public DateOnly? Birthday { get; set; }

    public string? Phone { get; set; }

    public virtual ICollection<Order> Orders { get; set; } = new List<Order>();
}

File: projectAngular/src/app/models/customer.ts
export class Customer {

  id: number;              // מזהה הלקוח
  fullName: string;       // שם מלא
  email: string;          // דוא"ל
  birthday: Date;         // תאריך לידה
  phone: string;          // טלפון

  constructor(
    id: number,
    fullName: string,
    email: string,
    birthday: Date,
    phone: string
  ) {
    this.id = id;
    this.fullName = fullName;
    this.email = email;
    this.birthday = birthday;
    this.phone = phone;
  }
}



